noir-projects/noir-contracts/contracts/test2_contract/src/main.nr

```diff
@@ -0,0 +1,8 @@
+contract Test2 {
+    use dep::aztec::state_vars::PublicMutable;
+
+    #[aztec(storage)]
+    struct Storage {
+        counters: PublicMutable<Field>,
+    }
+}
```

Comment:
Is this a remnant from when testing the macros?

Reply:
Yes, sorry forgot to cleaned that up. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/7437/commits/89f539560297a33544f150f2bef55faeed4c277b